README.md

```diff
@@ -31,55 +31,32 @@ The EventFlow suite supports .NET applications and .NET Core applications. It al
 The core of the library, as well as inputs and outputs listed above [are available as NuGet packages](https://www.nuget.org/packages?q=Microsoft.Diagnostics.EventFlow).
 
 ## Getting Started
-1. To quickly get started, you can create a simple console application in VisualStudio and install the Nuget package Microsoft.Diagnostics.EventFlow. 
-2. After the nuget package is installed, there should be a eventFlowConfig.json file added to the project. This file contains default configuration for the EventFlow pipeline. A few sections with optional configuration elements are commented outs. These sections can be enabled as desired. Here is what the file looks like
+1. To quickly get started, you can create a simple console application in VisualStudio and install the following Nuget packages:
+    * Microsoft.Diagnostics.EventFlow.Inputs.Trace
+    * Microsoft.Diagnostics.EventFlow.Outputs.ApplicationInsights
+    * Microsoft.Diagnostics.EventFlow.Outputs.StdOutput
+
+2. Add a JSON file named "eventFlowConfig.json" to your project and set the Build Action property of the file to "Copy if Newer". Set the content of the file to the following:
```

Comment:
You said it doesn't quite work, like what? The suite package doesn't work .net core, but in updated doc, the Trace input still doesn't work on .net core?

Installing 3 packages versus 1, plus add the config file, copy and paste, set build action to copy if newer, that's 5 more steps for the user. No matter how possible it is, there will be a higher chance of failure. Is there a strong reason behind that we want to remove this suite package?

Reply:
@yantang-msft newer versions of NuGet do not support adding anything other than binaries to the project. Neither the `eventFlowConfig.json` will be added, nor the included PowerShell script (that is supposed to ensure `eventFlowConfig.json` is copied to the output directory) will be executed. So the suite package does not add any value other than reducing the number of packages from 3 to 1.

Reply:
That's a huge breaking change! I can't find the doc, could you point me to it?